__test__/schema-inmutable.spec.ts

```diff
@@ -0,0 +1,119 @@
+import { getDefaultInstance, IManyQueryResponse, model, Schema } from '../src';
+import { delay, startInTest } from './testData';
+
+describe('Test Schema Immutable', () => {
+  const CardSchemaBase = new Schema({
+    cardNumber: { type: String, immutable: true },
+    zipCode: String,
+  });
+  const cardInfo = {
+    cardNumber: '5252 5252 5252 5252',
+    zipCode: '52525',
+  };
+  const cardInfoUpdate = {
+    cardNumber: '4242 4242 4242 4242',
+    zipCode: '42424',
+  };
+
+  test("Test Schema Immutable integration on strict=false'", async () => {
+    const CardSchema = new Schema(CardSchemaBase, { strict: false });
+    const Card = model('Card', CardSchema);
+    await startInTest(getDefaultInstance());
+    const result = await Card.create(cardInfo);
+    result.cardNumber = '80';
+    expect(result.cardNumber).toBe('80');
+  });
+
+  test("Test Schema Immutable integration on strict='throw'", async () => {
+    const CardSchema = new Schema(CardSchemaBase, { strict: 'throw' });
```

Comment:
What is strict: "throw" strict is a boolean field, Maybe you want to use strategy: CAST_STRATEGY. THROW